data/fucci-annotation-description.txt

```diff
@@ -25,4 +25,4 @@ reads	verifyBamID: The number of sequences that overlapped SNPs
 avg_dp	verifyBamID: The average sequencing depth that covered a SNP
 min_dp	verifyBamID: A minimun depth threshold for QC only (affects snps_w_min)
 snps_w_min	verifyBamID: The number of SNPs that had the minimum depth (min_dp); QC only
-valid_id	verifyBamID: Is the predicted individual 1 of the 4 added to the C1 chip?
+valid_id	verifyBamID: Is the predicted individual 1 of the 2 added to the C1 chip?
```

Comment:
This file is automatically created by [code/create-expressionset.R](https://github.com/jdblischak/fucci-seq/blob/b8c62c86afb45ab4d2edf0a3b15cfbf712e7f4c4/code/create-expressionset.R#L94). In the future, please make your changes directly to that file. I'll update it now.